docs/apis/cloud-events/cloud-events-overview.md

```diff
@@ -0,0 +1,17 @@
+---
+id: cloud-events-overview
+title: Cloud Events Overview 
+sidebar_label: Overview 
+description: "This section provides an overview of the different Cloud Event commands which are supported by Miranum Connect."
+---
+# Overview
+
+CloudEvents is a [specification](https://github.com/cloudevents/spec/blob/main/cloudevents/spec.md) for describing event 
```

Comment:
Would wrap this explanation in an info-box, since it's not really miranum specific.

Reply:
Great idea! 👍